src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_skadi.cpp

```diff
@@ -60,64 +58,66 @@ enum Spells
     SPELL_SUMMON_HARPOON                          = 56789,
 };
 
-enum Yells
+enum Texts
 {
-    SAY_AGGRO                           = 0,
-    SAY_KILL                            = 1,
-    SAY_DEATH                           = 3,
-    SAY_DRAKE_DEATH                     = 5,
-    SAY_DRAKE_BREATH                    = 6
+    SAY_AGGRO        = 0,
+    SAY_KILL         = 1,
+    SAY_DEATH        = 3,
+    SAY_DRAKE_DEATH  = 5,
+    SAY_DRAKE_BREATH = 6,
+    EMOTE_BREATH     = 0,
+    EMOTE_ON_RANGE   = 1
 };
 
-enum Emotes
-{
-    EMOTE_BREATH                        = 0,
-    EMOTE_ON_RANGE
-};
-
-enum Data
+enum Points
 {
-
-    DATA_LOVE_TO_SKADI                   = 0,
-    FIRST_WAVE_MAX_WARRIORS              = 10,
-    FIRST_WAVE_SIZE                      = 13,
-    GRAUF_PATH_INITIAL                   = 2689300,
-    GRAUF_PATH_RIGHT                     = 2689301,
-    GRAUF_PATH_LEFT                      = 2689302,
-    ACHIEV_LODI_DODI_WE_LOVES_THE_SKADI  = 17726,
+    POINT_0          = 0,
+    POINT_1          = 1,
+    POINT_2          = 2,
+    POINT_BREACH     = 0,
+    POINT_LEFT       = 1,
+    POINT_RIGHT      = 2
 };
 
-enum Points
+enum SplineChainIds
 {
-    POINT_0                              = 0,
-    POINT_1                              = 1,
-    POINT_9                              = 9,
-    POINT_11                             = 11,
-    POINT_LEFT                           = 21,
-    POINT_RIGHT                          = 22
+    SPLINE_CHAIN_INITIAL      = 1,
+    SPLINE_CHAIN_BREACH_RIGHT = 2,
+    SPLINE_CHAIN_BREACH_LEFT  = 3,
+    SPLINE_CHAIN_RIGHT        = 4,
+    SPLINE_CHAIN_LEFT         = 5
 };
 
 enum Actions
 {
-    ACTION_START_ENCOUNTER               = 0,
-    ACTION_FLAME,
+    ACTION_START_ENCOUNTER = 0,
+    ACTION_DRAKE_BREATH,
     ACTION_GAUNTLET_END,
     ACTION_HARPOON_HIT,
 };
 
 enum CombatPhase
 {
-    PHASE_FLYING                         = 0,
+    PHASE_FLYING = 0,
     PHASE_GROUND
 };
 
-Position const BreachPoint = { 0.0f, 0.0f, 0.0f, 2.670354f };
+enum MiscData
+{
+
```

Comment:
Unneeded blank line.
It will have to wait until next time the script is updated.